libs/sdk-backend-spi/src/workspace/attributes/elements/index.ts

```diff
@@ -39,6 +91,12 @@ export interface IElementsQueryOptions {
      * Include the total count of all elements in the response (without filters applied)
      */
     includeTotalCountWithoutFilters?: boolean;
+
+    /**
+     * Specify particular elements to load.
+     * This is commonly used to preload selected elements in the attribute filter.
```

Comment:
The second line of the comments should be in the remarks tag, it says some detailed info about the property